### DISCUSSION TITLE
Laravel SEO package with Vite

### DISCUSSION THREAD

**Discussion by shruko** [romanzipp/Laravel-SEO#51] [General]
Hi there,

I'm interested in using the Laravel SEO package with Vite instead of Laravel Mix. 
Can you provide any guidance or resources on how to do so? Is there any configuration required in the Laravel SEO package to make it work with Vite? 

Thanks in advance for your help!

**Comment by romanzipp** [comment]
I haven't used Vite in a Laravel project yet - only on Nuxt/Next. Could you provide an example repo?
